src/components/posts/PostsPage.js

```diff
@@ -4,68 +4,74 @@ import PostForm from './PostForm';
 import client from '../../utils/client';
 import './style.css';
 import { Box, Stack } from '@mui/material';
+import Avatar from '@mui/material/Avatar'
```

Comment:
I'm a bit confused here - shouldn't we be using the same icons for the profile image everywhere?